doc/source/releases/19.05.rst

```diff
@@ -212,8 +212,7 @@ Enhancements
 * Improve tool testing for counting datasets in paired collections
   (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
   `Pull Request 7359`_
-* Do not default to ``DEV_WHEELS=1`` if no ``git`` is found
-  (thanks to `@ic4f <https://github.com/ic4f>`__).
```

Comment:
@jdavcs I can restore these with the updated username, if that's OK?

If you'd like to be mentioned in the release notes for future PRs, I can remove your username from https://github.com/galaxyproject/galaxy/pull/13901/files#diff-5088adec198bd0b4206ec7080b366c9b0547edb62c3310e68b2b80f8eded6731R41 , let me know!

Reply:
That would be great, thank you!

Reply:
Done in a rebase!

Reply:
How do we actually want to manage that list?  As the comment states it is indeed a bit arbitrary at this point, but generally folks who are part of the core active contributing group are listed.

Reply:
Actually now that I think of it, since all (mostly all?) committers are not explicitly mentioned, I think it would make sense to  make it consistent across all committers. Or drop it altogether - no preference here, as long as it's consistent for everyone. @nsoranzo I'm sorry, I got confused and didn't realize that was the committer exclusion list, so if we keep that list, I'd like to be on it.

Reply:
I think this PR goes in the right direction (i.e. dropping the list), but maybe that could be a specific follow up PR? Happy to open it for discussion, if you agree.

Reply:
Yes, of course, no need to hold this up! Let's get this in and then we can have another one.

Reply:
Thanks, I've opened https://github.com/galaxyproject/galaxy/pull/13915 .